### PR TITLE
Feature/starting page

### PR DESCRIPTION
**Pull request by Stivaros**

### CODE REVIEW

File: spec/examples.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+example_id                                                    | status | run_time        |
```
Comment (garytaylor) [inline spec/examples.txt]: Not sure what this file is for and if it should be in VCS ?

File: spec/features/access_start_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.feature "Access Start Page" do
```
Comment (garytaylor) [inline spec/features/access_start_page_spec.rb]: can you pls make sure there is a ticket in jira to sort out the start page ?

File: app/assets/stylesheets/elements.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+@import 'elements/helpers';
+@import 'elements/reset';
+
+// Base (unclassed HTML elements)
+// These are predefined by govuk_template
+// If you're not using govuk_template, uncomment the line below.
+// HTML elements, set by the GOV.UK template
+// @import 'elements/base'
+
+@import 'elements/layout';
```
[the remark this answers is not in the document]
Comment (Stivaros) [reply app/assets/stylesheets/elements.scss]: Thanks, have commented out the elements that we're not using at this time.

File: bin/byebug [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```
[the remark this answers is not in the document]
Comment (Stivaros) [reply bin/byebug]: Found the commit they appeared in `4874163 Use nulldb method for forms` and reverted the bin changes.

File: spec/examples.txt (hunk 1 above)
Comment (Stivaros) [reply spec/examples.txt]: Nor I, it only appeared after I renamed the `features` folder. Added to `.gitignore` and untracked.

File: spec/features/access_start_page_spec.rb (hunk 2 above)
Comment (Stivaros) [reply spec/features/access_start_page_spec.rb]: Yes, this test fails and there is a ticket - `RST-867`.

**Pull request merged**